benchmarks/ImageProcessing/CMakeLists.txt

```diff
@@ -117,3 +117,29 @@ target_link_libraries(image-processing-benchmark
   # Link Buddy MLIR DIP Library.
   BuddyLibDIP
   )
+
+  #-------------------------------------------------------------------------------
```

Comment:
Formatting is inconsistent.

Reply:
Corrected the following formatting problem with _tab_